ansible/group_vars/nimbus.mainnet.yml

```diff
@@ -6,42 +6,118 @@ bootstrap__extra_volume_bind_path: null
 # SWAP
 swap_file_size_mb: 2048
 
-# Go-Ethereum
-geth_service_name: 'geth-{{ geth_network_name }}'
+# Ports used for all 3 EL nodes as they are mutually exclusive.
+exec_layer_p2p_port:     '{{ 30303 + 3*idx|int + 1 }}'
+exec_layer_rpc_port:     '{{ 8545  +   idx|int + 1 }}'
+exec_layer_authrpc_port: '{{ 8551  +   idx|int + 1 }}'
+exec_layer_metrics_port: '{{ 6060  +   idx|int + 1 }}'
+
+# Go-Ethereum ------------------------------------------------------------------
+
+geth_service_name: 'geth-{{ geth_network_name }}-{{ node.branch | mandatory }}'
 geth_service_path: '/docker/{{ geth_service_name }}'
 geth_network_name: 'mainnet'
 geth_cont_name: '{{ geth_service_name }}-node'
 geth_cont_vol: '{{ geth_service_path }}/node'
 geth_sync_mode: 'snap'
 geth_log_level_name: 'info'
+# Geth auth & JWT token
 geth_account_pass: '{{lookup("bitwarden", "nimbus/geth", field="password")}}'
-geth_authrpc_jwtsecret: '{{lookup("bitwarden", "nimbus/jwt-token")}}'
-# Memory settings¬
+geth_authrpc_jwtsecret: '{{ beacon_node_exec_layer_jwt_secret }}'
+# Memory settings
+# TODO: keep or reduce?
 geth_cont_mem_ratio: 0.15
 geth_cache_size: '{{ (ansible_memtotal_mb * 0.05|float) | int }}'
 # Ports
-geth_port: 30303
-geth_rpc_addr: '0.0.0.0'
-geth_rpc_port: 8545
-geth_authrpc_addr: '127.0.0.1'
-geth_authrpc_port: 8551
-# Geth metrics¬
+geth_rpc_addr:        '0.0.0.0'
+geth_authrpc_addr:    '127.0.0.1'
+geth_port:            '{{ exec_layer_p2p_port }}'
+geth_rpc_port:        '{{ exec_layer_rpc_port }}'
+geth_authrpc_port:    '{{ exec_layer_authrpc_port }}'
+geth_metrics_port:    '{{ exec_layer_metrics_port }}'
+# Geth metrics
 geth_expo_service_name: '{{ geth_service_name }}'
 geth_expo_source_cont_name: '{{ geth_cont_name }}'
 geth_expo_source_data_path: '{{ geth_cont_vol }}/data'
-geth_expo_cont_port: 9400
+geth_expo_cont_port:  '{{ 9400 + (idx|int) + 1 }}'
+
+# Erigon -----------------------------------------------------------------------
+
+erigon_network_name: 'holesky'
+erigon_service_name: 'erigon-{{ geth_network_name }}-{{ node.branch | mandatory }}'
+erigon_prune: 'hrtc' # TODO: Possibly wrong, verify.
+erigon_log_level: 'info'
+erigon_cont_mem_ratio: 0.15
+erigon_max_peers: 20
+erigon_miner_enabled: true
+# TODO: whats that?
+erigon_mining_private_key: '{{lookup("bitwarden", "nimbus/wallet/testnets", field="private-key")}}'
```

Comment:
I don't think this actually matters anymore, but in the past - before The Merge and transitioning to proof-of-stake - when just the EL was running and it was making all the money this was the wallet to get the earnings. I don't think this has any effect currently, but you can verify.

Reply:
Yeah, this shouldn't matter.